src/AspNet.Security.OAuth.SuperOffice/SuperOfficeAuthenticationConfiguration.cs

```diff
@@ -0,0 +1,63 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Text.Json.Serialization;
+
+namespace AspNet.Security.OAuth.SuperOffice
+{
+    /// <summary>
+    /// Contains well-known configuration settings.
+    /// </summary>
+    [System.Diagnostics.CodeAnalysis.SuppressMessage("Usage", "CA2227:Collection properties should be read only", Justification = "Required for serialization.")]
+    public class SuperOfficeAuthenticationConfiguration
+    {
+        public SuperOfficeAuthenticationConfiguration()
+        {
+        }
+
+        [JsonPropertyName("issuer")]
+        public string Issuer { get; set; } = string.Empty;
+
+        [JsonPropertyName("authorization_endpoint")]
+        public string AuthorizationEndpoint { get; set; } = string.Empty;
+
+        [JsonPropertyName("token_endpoint")]
+        public string TokenEndpoint { get; set; } = string.Empty;
+
+        [JsonPropertyName("end_session_endpoint")]
+        public string EndSessionEndpoint { get; set; } = string.Empty;
+
+        [JsonPropertyName("jwks_uri")]
+        public string JwksUri { get; set; } = string.Empty;
+
+        [JsonPropertyName("scopes_supported")]
+        public ICollection<string> Scopes { get; set; } = new Collection<string>();
```

Comment:
Could just use a standard `List<string>` for these properties, as it implements both `IList<T>` and `ICollection<T>`, meaning you then don't need to use the `System.Collections.ObjectModel` namespace.

Reply:
Change hasn't been made.